src/routes/safe/components/AddressBook/index.tsx

```diff
@@ -243,6 +243,7 @@ const AddressBookTable = (): ReactElement => {
         isOpen={sendFundsModalOpen}
         onClose={() => setSendFundsModalOpen(false)}
         recipientAddress={selectedEntry && selectedEntry.entry ? selectedEntry.entry.address : undefined}
+        recipientName={selectedEntry && selectedEntry.entry ? selectedEntry.entry.name : undefined}
```

Comment:
this now can be replaced with `selectedEntry?.entry?.name` isn't it?

same for `recipientAddress`